sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -363,6 +364,27 @@ export function useFormatters() {
     }
   };
 
+  /**
+   * Convert the application disability status to be displayed
+   * in an equivalent user friendly description.
+   * @param disabilityStatus disability status.
```

Comment:
The param name must be adjusted from `disabilityStatus` to `applicationDisabilityStatus`.